functions/v2/helloAuditLog/.gcloudignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
Should this be done in `functions/v2/` instead?

(I'm not sure if `gcloud` checks parent directories.)

Reply:
**Update:** @saraford confirmed that the `.gcloudignore` needs to the in the _function's_ root directory.

(We can ignore this comment.)